content/blog/2018/06/2018-05-14-simple-pull-request-plugin.adoc

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "GSoC Project Intro: Simple pull request Plugin"
+tags:
+- gsoc2018
+- plugin
+author: abhishek_gautam
+---
+
+=== About me
+
+I am Abhishek Gautam, 3rd year student from Visvesvaraya National Institute of technology, India, Nagpur.
+
+As a person who is passionate is intrested in automation.
+
+=== Project Summary
+This plugin will be developed on the top of the MultiBranch Pipeline plugin.
+
+.This plugin will follow below steps
+* clone target repo
+* checkout to target branch
+* fetch the source branch
+* merge source-branch
+* call user call user script to build the repo.
+* push to target branch [if specified by user in YAML]
+* harvest results and reports and post to PR
+
+Plugin will start above steps if and only if the pull request is will be
+mergeable to avoid merge conflicts while merging the source branch to target
+branch. The pull request is mergebale or not can also be decided by the webhook also.
```

Comment:
I do not understand what this sentence means: "The pull request... by the webhook also".

Reply:
It should be "the payload of webhook".